packages/app/src/specs/SpecListGitInfo.vue

```diff
@@ -1,24 +1,34 @@
 <template>
-  <div class="h-full grid gap-9px grid-cols-[16px,auto] git-info-row items-center">
-    <span
-      class="flex w-16px items-center justify-center"
-      :data-cy="`git-status-${props.gql.statusType}`"
+  <div
+    class="h-full grid gap-9px grid-cols-[16px,auto] git-info-row items-center"
+  >
+    <Tooltip
+      placement="top"
+      class="h-full grid items-center"
+      :skip-tooltip="!classes.showTooltip"
     >
-      <span
-        v-if="props.gql.statusType !== 'unmodified'"
-        class="rounded-full border-1 min-w-6px min-h-6px max-w-6px max-h-6px"
-        :class="classes?.indicator"
+      <component
+        :is="classes.icon"
+        :class="classes?.iconClasses"
```

Comment:
Is optional chaining necessary? From the definition of `classes` on line 163, it looks like it will always be an object?